test/nuts/ebikes-lwc/.eslintignore

```diff
@@ -0,0 +1,13 @@
+**/lwc/**/*.css
```

Comment:
why do we need a clone of ebikes in the repo? Why can't we clone the project for NUTs?

Reply:
I expect the sample apps to change over time.  Tracking nuts are (perhaps overly-) specific about how many rows there should be in various scenarios.